RenderStateNotation/scripts/cxx_parser.py

```diff
@@ -42,78 +42,90 @@
 CXX_PRAGMA_ONCE = "#pragma once"
 CXX_NAMESPACE = "namespace {}"
 
-CXX_PATTERN_INTERFACE = r"(Diligent::I[A-Z].*[a-z])|(struct I[A-Z].*[a-z])"
-CXX_PATTERN_STRING = r"const(\s)+(char|Diligent::Char)(\s)+\*$"
+CXX_PATTERN_INTERFACE = r"(I[A-Z].*[a-z])|(struct I[A-Z].*[a-z])"
```

Comment:
I think this pattern may be too general and catch unrelated objects. Similar problem already happened a few times

Reply:
Currently, this is functioning. In theory, an additional search for the corresponding GUID could be added to ensure that it's a COM interface, but I believe it's better to create a separate task for that purpose.